fc-exploitation/src/service-provider/dto/service-provider-input.dto.ts

```diff
@@ -86,8 +86,8 @@ export class ServiceProviderDto {
   readonly scopes: string[];
 
   @IsOptionalExtended()
-  @IsIn([ES256, RS256, HS256], {
-    message: `<strong>userinfo_signed_response_alg</strong> doit être une des valeurs suivantes: ${ES256}, ${RS256} ou ${HS256}`,
+  @IsIn([EMPTY, ES256, RS256, HS256], {
```

Comment:
suggestion: your implementation is perfectly valid but...

Most of the transformations on input are done here instead of in https://github.com/proconnect-gouv/federation-admin/pull/14/files#diff-a63c17ea4583c28c06b6a9f15d8804c92d2c3a0be1d656b7ef5126dc61131b5dR107 

As an illustration, in src/service-provider/service-provider.service.ts, we default the claims on the update operation and not on the create. This could have been avoided by using a single transformation in this file.

I suggest we keep this direction for now.

Reply:
J'aime bien, nonobstant je ne vois pas dans federation-admin d'endroit où ce décorateur @Transform est appelé (alors que dans federation c'est le plainToInstance qui est appelé à chaque fois). Tu saurais me pointer comment faire ça ?